python/python/lancedb/query.py

```diff
@@ -1605,6 +1610,7 @@ def __init__(self, inner: LanceQuery):
         """
         super().__init__(inner)
         self._inner = inner
+        self._reranker = RRFReranker()
```

Comment:
I'm not sure this is needed. The reranker can be a member of the query builder for the concrete search type (vector, FTS or Hybrid)